src/MainApp.jsx

```diff
@@ -37,6 +38,8 @@ const MainApp = () => (
         <Redirect to={updatePathWithQueryParams(REGISTER_PAGE)} />
       </Route>
       <UnAuthOnlyRoute exact path={LOGIN_PAGE} render={() => <Logistration selectedPage={LOGIN_PAGE} />} />
+      { !allowPublicAccountCreation
```

Comment:
can we use the identifier `disablePublicAccountCreation` instead of `allowPublicAccountCreation` which will increase readability in my opinion. but its up to you.